### PR TITLE
Message buttons

### PR DESCRIPTION
**Pull request by huwd**

### CODE REVIEW

File: src/Component/Atom/MessageButton/MessageButton.jsx [hunk 1]
```diff
@@ -18,34 +18,47 @@ const MessageButtonInner = styled.a`
   width: 24rem;
   font-size: 1.4rem;
   font-weight: 700;
-  background-color: #ed1e79;
-  border: .2rem solid #d44583;
+  ${(props) => (console.log(props.disabled))}
```
Comment (Gisleburt) [inline src/Component/Atom/MessageButton/MessageButton.jsx]: Left over `console.log`?
eslint should have picked up on that 🤔

File: src/Component/Atom/MessageButton/MessageButton.jsx [hunk 2]
```diff
@@ -18,34 +18,47 @@ const MessageButtonInner = styled.a`
...
-  }
+  ${(props) => (
+    props.disabled ? ''
+      : `
+        :hover {
+          background-color: #fff;
+          border-color: #ed1e79;
+          color: #ed1e79;
+        }
+      `)
```
Comment (Gisleburt) [inline src/Component/Atom/MessageButton/MessageButton.jsx]: Something feels wrong about the spacing here, what does eslint say?

File: src/Component/Atom/MessageButton/MessageButton.jsx [hunk 3]
```diff
@@ -18,34 +18,47 @@ const MessageButtonInner = styled.a`
...
 `;
 
+
 const MessageButton = ({ to, from }: Props) => {
-  if (to && from && canMessageMember(to, from)) {
+  if (to && canMessageMember(to, from)) {
+    console.log("HELLO");
     return (
-      <MessageButtonInner href={`/members/${from.slug}/messages/compose/?r=${to.slug}`}>
+      <MessageButtonInner href={!from ? '/' : `/members/${from.slug}/messages/compose/?r=${to.slug}`} disabled={!!from}>
```
Comment (Gisleburt) [inline src/Component/Atom/MessageButton/MessageButton.jsx]: Reverse this ternary to get rid of the unnecessarey not (`!`)

File: src/Component/Atom/MessageButton/MessageButton.jsx (hunk 1 above)
Comment (huwd) [reply src/Component/Atom/MessageButton/MessageButton.jsx]: Sorry this is a misunderstanding, this isn't ready for reivew, just meant I'd push up my current branch - thought I'd tag it WIP for visibility

File: src/Component/Atom/MessageButton/MessageButton.jsx (hunk 2 above)
Comment (huwd) [reply src/Component/Atom/MessageButton/MessageButton.jsx]: actually that one got past the linter, weird no?

File: src/Component/Atom/MessageButton/MessageButton.jsx (hunk 3 above)
Comment (huwd) [reply src/Component/Atom/MessageButton/MessageButton.jsx]: yeah none of it works anyway

File: src/Component/Atom/MessageButton/MessageButton.jsx [hunk 4]
```diff
@@ -17,39 +17,67 @@ const MessageButtonInner = styled.a`
...
   color: #fff;
 
   text-decoration: none;
 
-  :hover {
-    background-color: #fff;
-    border-color: #ed1e79;
-    color: #ed1e79;
-  }
+  ${(props) => (
```
Comment (huwd) [inline src/Component/Atom/MessageButton/MessageButton.jsx]: This is starting to get smell IMO,
Debating two alternative ways to slice this, either:
- push this functionality into the CTA button and actually use it here, OR
- refactor this out into multiple components and simplify the render section
- or... de-smell these props?

**Pull request merged**